mods/ra/weapons/other.yaml

```diff
@@ -177,6 +208,22 @@ TTankZap:
 			None: 1000
 		DamageTypes: Prone50Percent, TriggerProne, ElectricityDeath
 
+TTankCharge:
+	ReloadDelay: 120
+	Range: 2c0
+	Report: TESLACHARGE.AUD
+	Charges: yes
+	ValidStances: Ally
+	ValidTargets: TeslaBoost
+	Projectile: TeslaZap
+		Image: litningupg
+	Warhead@charge: GrantUpgrade
+		Range: 42
+		Duration: 125
+		Upgrades: tesla, tesla1, tesla2
```

Comment:
Because of timed upgrades again...
Only one levelup is permitted per timed upgrade source. This is OK for shock troopers since they only give one tesla up each, but it is a little problematic for tesla tanks since they need to give three tesla ups (obviously all with the same source).

Simply allowing more than one levelup per timed upgrade source is kinda problematic - take a shock trooper that grants a 75 tick upgrade every 70 ticks. If you allow multiple upgrades per source, then for a brief period of 5 ticks, the shock trooper would give two tesla ups to the tesla coil. If you make the shock trooper grant a 70 tick upgrade every 70 ticks, then the upgrade level can at times actually drop to 0 on the 70th tick (which causes e.g. that the tesla coil loses its target and other weird stuff).

But yeah, this isn't too pretty. Maybe it'd be nicer if tesla tanks gave a different kind of upgrade, that would count as three shock trooper upgrades?